### PR TITLE
add v3.11 ceph-csi manifests

### PR DESCRIPTION
**Pull request by kwmonroe**
Adding v3.11 (needed for k8s 1.26+).  We won't switch this to being the default until canonical k8s integrations are complete / verified.

- add manifests for v3.11
- set default to 3.9 (currently supported)
- tweak action name and LICENSE

### CODE REVIEW

**Comment by addyess** [comment]
@kwmonroe i fixed your unit/linting tests -- they started failing though.
I bet its something easy

File: config.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ options:
 
   release:
     type: string
+    default: "v3.9.0"
```
Comment (addyess) [inline config.yaml:53]: Hmm... i'm not sure i like defaulting here.  You could just leave v3.9.0 in the `version` file right?  That should update the defacto-default without changing the charm config right?

What's the intent maybe i'm missing it

Comment (kwmonroe) [reply config.yaml:53]: @addyess i felt like `version` was too easy to accidentally bump.  any time you update.py to bring in new manifests, `version` is reset.  i liked config as a more explicit place to say "this charm is known to work with this version" as well as being able to unset the config to say "just give me the latest".

i don't feel too strongly either way, but that was my train of thought.  if you'd prefer using `version` for the default config, i can get behind that.

Comment (addyess) [reply config.yaml:53]: nah, that's a good enough reason for me!  Someone can config this value to adjust releases!
👍🏻

**Review by addyess — APPROVED** [review]
Way to go, thanks for adding all these new versions and testing with ceph

**Pull request merged**